### ISSUE TITLE
NVDA doesn't switch focus back to ComboBox after its dropdown is closed

### ISSUE THREAD

**Issue by dmitrii-drobotov** [dotnet/winforms#8218]
### .NET version

.NET 7

### Did it work in .NET Framework?

No

### Issue description

After expanding the ComboBox and selecting value in dropdown, NVDA still keeps focus on the dropdown and doesn't switch back to the ComboBox edit field. As a result, when changing ComboBox value by arrows, NVDA doesn't announce it.

It was originally found in https://github.com/dotnet/winforms/issues/7763 comments, and there is more information with NVDA logs and recordings.

### Steps to reproduce

1. Launch NVDA screen reader
2. Expand a combo box with keyboard (alt + down shortcut)
3. Select an element from dropdown list, press Enter or Esc to close the dropdown — NVDA highlight rectangle still stays on the dropdown item
4. Use up/down arrows to change selected element — NVDA is not announcing the change

**Comment by MelonWang1** [comment]
Verified on latest 8.0 build .NET 8.0.100-alpha.1.22607.6, issue is fixed. NVDA has switch focus back to ComboBox after its dropdown is closed. But when close dropdown and Press Down: change selection to item "3", Press Down: change selection to item "4", the Speech Viewer don't show 3, 4. Gif as below:
![8220-1](https://user-images.githubusercontent.com/94418985/206350282-a0dbbbd6-8a1c-48d0-bafb-a773fc8be7ca.gif)

**Comment by dmitrii-drobotov** [comment]
@MelonWang1 Could you please create a new issue for this bug?

**Comment by MelonWang1** [comment]
@dmitrii-drobotov OK, thank you! I filed a new issue [8395](https://github.com/dotnet/winforms/issues/8395) to track this bug.
